packages/terra-folder-tree/src/subcomponents/FolderTreeItem.jsx

```diff
@@ -169,23 +196,16 @@ const FolderTreeItem = ({
         className={itemClassNames}
         role="treeitem"
         aria-expanded={isFolder ? isExpanded : null}
-        aria-selected={isSelected}
-        onClick={isFolder ? handleToggle : onClick}
+        aria-selected={isSelectable ? isSelected : null}
```

Comment:
According to the [mdn docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-selected#values), the states for `aria-selected` are **true**, **false**, and **undefined** (the element is not selectable). I'd like to avoid setting `aria-selected={false}` on elements that are not selectable (although it does seem to behave correctly in Voiceover at least) since it implies that the element is selectable.